### PR TITLE
Serum JS guide

### PR DESCRIPTION
**Pull request by philippe-ftx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,16 +19,64 @@ Using yarn:
 yarn add @solana/web3.js @project-serum/serum
 ```
 
-## Usage
+# Guide
+
+This guide will cover every aspect of Serum-JS and how it can be used to interact with the Serum DEX. For a technical introduction to the Serum DEX, please take a look at: https://docs.google.com/document/d/1isGJES4jzQutI0GtQGuqtrBUqeHxl_xJNXdtOv4SdII
+
+To interact with the DEX, a Wallet adapter is needed: https://github.com/project-serum/sol-wallet-adapter.
```
Comment (garywang) [inline README.md:26]: This is only true for integrating the dex into a web page. The typical user would use an `Account` directly.

**Comment by garywang** [comment]
We should also document the `placeOrder` and `cancelOrder` methods since they're more convenient to use than `makePlaceOrderTransaction` and `makeCancelOrderTransaction`.

**Comment by garywang** [comment]
Also we should document `cancelOrderByClientId`

**Comment by armaniferrante** [comment]
This repository has moved to https://github.com/project-serum/serum-ts/tree/master/packages/serum. Please migrate this PR over, if desired.

**Pull request closed without merging**